lib/trento/infrastructure/router.ex

```diff
@@ -26,7 +23,7 @@ defmodule Trento.Router do
   }
 
   middleware Enrich
-  middleware Validate
+  # middleware Validate
```

Comment:
Do we wanna remove this?

Reply:
yes we want

Reply:
there is no need to validate since the structs are already validated